tests/test_config.py

```diff
@@ -38,7 +38,7 @@ def _config(value):
 for ctype in ("env", "cfg"):
     if ctype == "cfg" and ENVKEY in os.environ:
         del os.environ[ENVKEY]
-    for mode in ("true", "false", "yes", "no", "on", "off", "default", ""):
+    for mode in ("true", "false", "yes", "no", "on", "off", "default"):
```

Comment:
There seem to be os-specific differences in behavior interpreting the blank string as the yaml config value. It's just not worth worrying about, as people are highly unlikely to use it